packages/esbuildnx/package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "generators": "./generators.json",
   "executors": "./executors.json",
+  "builders": "../../@nrwl/node/builders.json",
```

Comment:
not very cool - but needed for us temporarily to be able to serve using the old tools. we will start looking into using esbuild for dev soon as well.

Reply:
Do you think that the builders should be something more like `node_modules/@nrwl/node/builders.json`? 
Or maybe it will always work this way?

Reply:
I'm not sure - preferably I'd like to have hot-reload and build the applications using esbuild, just like the current node builders. 

We tried `"builders": "./executors.json",` as well. There seems to be some differences between the executors in esbuildnx and the builders in node/builders wrt what they yield to nx.

Reply:
But, I did put in a watch for build. You can always set up a serve and run build on a watch. That's part of the way to hot reload. 

I published this anyway, but I think that it is worth making the copy asset part of the esbuildnx package without importing it from   @nrwl. Or at least, a better import method. I wish that the nrwl team would properly export more of their tooling functions.

Reply:
We will be taking a better look at the watch thing - we just needed this temporary fix asap to avoid blocking developers.

WRT the copy asset part I agree that the best course of action would be to be able to import it, mostly because we would always want the transition from @nrwl/node to esbuildnx to be as smooth and painless as possible.